contents/newsletter/choosing-technologies.md

```diff
@@ -0,0 +1,115 @@
+---
+title: What we've learned about choosing technologies
+date: 2024-10-29
+author:
+ - ian-vanagas
+featuredImage: >-
+featuredImageType: full
+tags:
+  - Engineering
+crosspost:
+  - Product engineers
+  - Founders
+  - Blog
+---
+
+From generating text to storing data to creating [ASCII drawings of cows](https://github.com/sindresorhus/cows), if you've got a problem, there is a technology out there to solve it for you.
+
+Technologies are the backbone of a product. They set the guidelines of what they can and cannot do. Choosing the right technologies makes a massive difference in whether you succeed or fail. 
+
+Unfortunately, there is no perfect way to pick a technology. Some adopt everything possible, others build anything they need themselves. Like many, we are somewhere in the middle. We aim to move fast and use technology to build something people want.
+
+Apparently, we're doing an ok job of this. Over the past year, we've shipped multiple new products like web analytics and mobile session replay on top of improving our existing products like pipelines and feature flags. Each comes with its own set of technologies. 
+
+This is what we've learned about choosing them all.
+
+## Start with a hair-on-fire problem burning problem
```

Comment:
Both this and the next section feel a bit short on their own. Consider merging into one.

Reply:
Merged